### PR TITLE
Add support for linux-zen kernel

### PR DESCRIPTION
**Pull request by hexchain**

### CODE REVIEW

File: src/kernels/linux-zen.sh [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# For build.sh
+mode_name="zen"
+mode_desc="Select and use the packages for the linux-zen kernel"
+
+# Kernel versions for default ZFS packages
+pkgrel="1"
+kernel_version="4.12.2-1"
```
Comment (minextu) [inline src/kernels/linux-zen.sh]: 4.12.2 is in testing right now, so start building this packages when this hits extra?

**Comment by minextu** [comment]
Closes https://github.com/archzfs/archzfs/issues/67

**Comment by hexchain** [comment]
Rebased, still waiting for linux-zen going into extra...

**Comment by demizer** [comment]
Should I wait for linux-zen to go into extra before I merge?

**Comment by hexchain** [comment]
Status of this?

**Comment by demizer** [comment]
Looks like linux-zen is in extra. Merging...

**Comment by hexchain** [comment]
I guess we should create {zfs,spl}-linux-zen{,-git} on AUR?

**Pull request merged**